examples/hello_world/gcp/config.textproto

```diff
@@ -0,0 +1,16 @@
+# proto-file: oak/proto/manager.proto
+# proto-message: oak.ApplicationConfiguration
```

Comment:
I think these comments are not necessary, since it's not a `oak/proto/manager.proto` file